WalletWasabi/Helpers/Constants.cs

```diff
@@ -115,5 +115,8 @@ public static class Constants
 		public static string ClientSupportBackendVersionText => ClientSupportBackendVersionMin == ClientSupportBackendVersionMax
 				? ClientSupportBackendVersionMin
 				: $"{ClientSupportBackendVersionMin} - {ClientSupportBackendVersionMax}";
+
+		public static readonly FeeRate MinRelayFeeRate = new FeeRate(1m);
+		public static readonly FeeRate AbsurdlyHighFeeRate = new FeeRate(10_000m);
```

Comment:
Does this mean that you can't set a fee rate higher thatn 10000 sat/vbyte?

@jmacato used something different here https://github.com/zkSNACKs/WalletWasabi/pull/4036/files#diff-51e6ce1a81598688a067e6be2361d7aeR65-R66, shouldn't we consider uniting them?

Reply:
I am aware of that value and it make no sense at all to me. What's the point on limiting the fee rate to the upper bound limit of satoshis in existance? Imagine I have a transaction that requires 1_000 sats/vbyte, the UI won't let me pay 21 millon bitcoins per byte, I mean, it protects me from paying 21 billion bitcoins in fee. It makes no sense.

The RPC is an interface that can be used for many different things and I think we should limit the chances of making huge mistakes. The rate 10_000 sats/vbyte could be too much restrictive and we can change it but i would prefer some useful limit.